build/compiled_action.gni

```diff
@@ -91,6 +91,8 @@ template("compiled_action") {
       pool = invoker.pool
     }
 
+    forward_variables_from(invoker, ["metadata"])
```

Comment:
You can use `forward_variables_from` for the three above too as a cleanup:
```suggestion
    forward_variables_from(invoker, ["metadata", "pool", "testonly", "visibility"])
```

Reply:
Very true! newest commit forwards all suitable variables.